src/common/utils.h.in

```diff
@@ -0,0 +1,54 @@
+/*!
```

Comment:
This seems like a strange file name.

Reply:
That is the original utils.h from h2o4gpu with some functions moved from gpu code.
For the *.h.in, it's for cmake configuration, specifically USE_CUDA flag. I know compiling without CUDA is not supported. But anyway I tried to remain the possibility. If that a burden, I can remove it in no time.